src/foundry/client-esm/dice/terms/dice.d.mts

```diff
@@ -356,21 +346,28 @@ declare namespace DiceTerm {
   }
 
   interface KeepOrDropOptions {
-    keep?: boolean | undefined;
-    highest?: boolean | undefined;
+    /** Keep results?  (default: `true`) */
+    keep?: boolean | undefined; // these cannot be null because they are compared in the function
```

Comment:
"Compared" is a bit vague. If you meant "cannot be `null` because it's checked against `=== undefined`" you can write that? I'd recommend either expanding the comment or going to `// not: null` or whatever.